src/amo/components/SearchResult/index.js

```diff
@@ -46,10 +46,22 @@ export class SearchResultBase extends React.Component<Props> {
 
     // Fall-back to default icon if invalid icon url.
     const iconURL = getAddonIconUrl(addon);
-    const themeURL = (addon && addon.themeData &&
-      isAllowedOrigin(addon.themeData.previewURL)) ?
-      addon.themeData.previewURL : null;
-    const imageURL = isTheme ? themeURL : iconURL;
+
+    let imageURL = iconURL;
+
+    if (isTheme) {
+      let themeURL = addon ? addon.previews
+      && addon.previews.length > 0
+      && addon.previews[0].thumbnail_url : null;
```

Comment:
I'd like to avoid the mix of ternary and logical operator. It's fine to use both, but not together. Here, a simple `if/else` would probably be more readable.

Reply:
hi @willdurand - are you talking about mixing in the same snippet of code? like line 53  here and line 63?